### ISSUE TITLE
Problem on v1.0.0-bata.0

### ISSUE THREAD

**Issue by gentarok** [react-bootstrap/react-bootstrap#3274]
1. Create new CRA app and add react-bootstrap module
```
create-react-app react-bootstrap-test
cd react-bootstrap-test
yarn add react-bootstrap@next
```

2. Modify App.js
```javascript
import React, { Component } from 'react';
import { Button } from 'react-bootstrap';

class App extends Component {
  render = () => (<Button>button</Button>)
}

export default App;
```

3. Run -> error
```
Failed to compile
./node_modules/react-bootstrap/es/DropdownMenu.js
Module not found: Can't resolve '@babel/runtime-corejs2/core-js/array/from' in '/home/foo/react-bootstrap-test/node_modules/react-bootstrap/es'
This error occurred during the build time and cannot be dismissed.
```

4. Add @babel/runtime-corejs2
```
yarn add -D @babel/runtime-corejs2
```

5. Run -> error
```
Failed to compile
./node_modules/react-bootstrap/es/Modal.js
Module not found: Can't resolve 'react-overlays/lib/Modal' in '/home/foo/react-bootstrap-test/node_modules/react-bootstrap/es'
This error occurred during the build time and cannot be dismissed.
```

6. Add react-overlays
```
yarn add react-overlays
```

Run -> success.

**Comment by jquense** [comment]
Looks liek there was a problem publishing, and the old code was released. pushing a new release now

**Comment by jquense** [comment]
fixed in 1.0.0-beta.1
